chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -370,9 +374,26 @@ impl PeerManagerActor {
         client_addr: Recipient<NetworkClientMessages>,
         view_client_addr: Recipient<NetworkViewClientMessages>,
         genesis_id: GenesisId,
+    ) -> anyhow::Result<Self> {
+        PeerManagerActor::new_with_clock(
+            store,
+            config,
+            client_addr,
+            view_client_addr,
+            genesis_id,
+            time::Clock::real(),
+        )
+    }
+
+    pub fn new_with_clock(
+        store: near_store::Store,
+        config: config::NetworkConfig,
+        client_addr: Recipient<NetworkClientMessages>,
+        view_client_addr: Recipient<NetworkViewClientMessages>,
+        genesis_id: GenesisId,
+        clock: time::Clock,
```

Comment:
please make clock the first argument. It is a convention that I'd like to introduce (originated in Golang :P).

Reply:
done.